unittests/test_highlevel.py

```diff
@@ -424,3 +426,98 @@ def test_extract_all_ebd_tables_from_v42(self, subtests):
                     pass  # Ignore known unsupported formats (e.g., "--" values)
                 except StepNumberNotFoundError:
                     pass  # Table format not supported (no valid step number found)
+
+
+class TestEbdDocumentReleaseInformation:
+    """
+    Tests for extracting release information from the title page of EBD documents.
+    """
+
+    @pytest.mark.parametrize(
+        "path, expected_version, expected_release_date, expected_original_release_date",
+        [
+            pytest.param(
+                EBD_2022_11_28,
+                "3.2",
+                date(2022, 11, 28),
+                date(2022, 4, 29),
+                id="v3.2 with correction (Ursprüngliches Publikationsdatum)",
+            ),
+            pytest.param(
+                EBD_2023_06_19_V33,
+                "3.3",
+                date(2023, 6, 19),
+                date(2022, 9, 30),
+                id="v3.3 with correction (Ursprüngliches Publikationsdatum)",
+            ),
+            pytest.param(
+                EBD_2023_06_29_V34,
+                "3.4",
+                date(2023, 6, 29),
+                date(2023, 3, 31),
+                id="v3.4 with correction (Ursprüngliches Publikationsdatum)",
+            ),
+            pytest.param(
+                EBD_2024_04_03_V35,
+                "3.5",
+                date(2024, 7, 31),
+                date(2023, 10, 4),
+                id="v3.5 with correction (Ursprüngliches Publikationsdatum)",
+            ),
+            pytest.param(
+                EBD_2025_04_04_V40B,
+                "4.0b",
+                date(2024, 10, 1),
+                date(2024, 10, 1),
+                id="v4.0b fresh release (Publikationsdatum equals Stand)",
+            ),
+            pytest.param(
+                EBD_V42,
+                "4.2",
+                date(2025, 12, 11),
+                date(2025, 10, 1),
+                id="v4.2 with correction",
+            ),
+        ],
+    )
+    def test_get_ebd_document_release_information(
```

Comment:
rewrite the test such that it compares against an `Optional[EbdDocumentReleaseInformation]` model instance (or null) instead of multiple dates and strings